### ISSUE TITLE
[Quest] Fortunate Misunderstandings

### ISSUE THREAD

**Issue by lastbullet00** [WoWManiaUK/Redemption#2188]
[http://www.wow-mania.com/armory/?quest=12570](url)

After getting the quest and talking to the NPC to start the escort part of quest he doesn't do anything, just stay there. I quit and retake the quest from all 5 NPC's in the area but none worked.

Other players said that it worked for them, the GM Anubix tried and said it works but still...maybe it have something with my character been a worgen, i while ago on other character (also a worgen) i had same problem.

![image](https://user-images.githubusercontent.com/34487562/35752715-a3715110-0865-11e8-8308-15e16ad84c76.png)
![image](https://user-images.githubusercontent.com/34487562/35752725-ad5be8de-0865-11e8-9d46-be99c3ac6128.png)

**Comment by goldstinger** [comment]
I have ran into this same problem too. I checked the bug listings for issues on the same quest and it has been reported multiple times, with one response being to contact a GM to fix this issue, from Janitor back in I think July of last year? Somewhere around there.

Chiming in to confirm that this is still an issue, and still requires a GM to complete the quest for people to progress past this part of the quest chain.
